modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -880,6 +1265,28 @@ private ObjectRelationship _addObjectRelationshipAndRelateObjectsEntries(
 		return objectRelationship;
 	}
 
+	private <T> void _assertFilter(
+			String expectedObjectFieldName, T expectedObjectFieldValue,
```

Comment:
Maybe better to use `Serializable expectedObjectFieldValue` instead to keep the same format as in the rest of the test?